src/debugging/TNSDebugging.h

```diff
@@ -428,6 +430,36 @@ static void TNSEnableRemoteInspector(int argc, char** argv,
           });
           CFRunLoopWakeUp(CFRunLoopGetMain());
         });
+   
+    int refreshRequestSubscription;
+    notify_register_dispatch(
+        NOTIFICATION("RefreshRequest"), &refreshRequestSubscription,
+        dispatch_get_main_queue(), ^(int token) {
+            @try
+            {
+                JSGlobalContextRef context = runtime.globalContext;
+                JSObjectRef globalObject = JSContextGetGlobalObject(context);
+                JSStringRef liveSyncMethodName = JSStringCreateWithUTF8CString("__onLiveSync");
+                JSValueRef liveSyncMethod = JSObjectGetProperty(context, globalObject, liveSyncMethodName, NULL);
+                JSStringRelease(liveSyncMethodName);
+                if (JSValueIsUndefined(context, liveSyncMethod))
+                {
+                    LOG_FAILED_REFRESH;
```

Comment:
You should return after logging the failure. Also, in order to make diagnostics easier you'd better log the reason for failure as well (possibly as a separate log message so that you won't have to handle it in the CLI's parser).